src/power_meter.cpp

```diff
@@ -0,0 +1,133 @@
+#include "power_meter.h"
+
+// TODOLater - Test with hardware
+PowerMeter::PowerMeter(uint8_t device_address, int pin_de, int pin_re)
+    : device_address_(device_address), pin_de_(pin_de), pin_re_(pin_re) {
+  node_.begin(device_address_, Serial);
+  pinMode(pin_re_, OUTPUT);
+  pinMode(pin_de_, OUTPUT);
+  digitalWrite(pin_re_, 0);
+  digitalWrite(pin_de_, 0);
+  // TODOLater - Decide if removing this is okay
+  //   node_.preTransmission(preTransmission);
```

Comment:
Based on the online code, except `node_.preTransmission()` requires a static function but `PowerMeter::preTransmission()` uses non-static variables. As such, it can't be used here
To get around this, `PowerMeter::preTransmission()` and `PowerMeter::postTransmission()` are being called individually when required. Not sure if this will introduce unwanted side effects though